install/langs.php

```diff
@@ -1,4 +1,4 @@
-<?php $langs = array (
+<?php $langs = [
```

Comment:
This file is generated via `CRM/Core/CodeGen/I18n.php`:

```php
file_put_contents('../install/langs.php', "<?php \$langs = " . var_export($langs, TRUE) . ";");
```

Reply:
A newline would be nice after the `<?php`

Reply:
I've removed it from the PR